package.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
```

Comment:
We might as well hardcode the APP= variable to the name we want since this script is quite specific to kubits due to all of the individual files specified as part of the package payload.